contrib/polybar.xbps/template

```diff
@@ -28,15 +27,25 @@ distfiles="
  https://github.com/jaagr/xpp/archive/${_xpp_version}.tar.gz
  $(vopt_if i3 https://github.com/jaagr/i3ipcpp/archive/v${_i3ipcpp_version}.tar.gz)"
 checksum="
- a6a25a5305b7357b11769de6c0f5bf3175d7c65f1c80615779e75cb22aa6892c
- b81993893190bf7ba19529e0da6e8b069e13e833f6a62874830b46785bee3349
- $(vopt_if i3 7ad9c0c1e1cb9dfd07ecc2a7769eae9ac962c22db2e07be08b2186b7fd85189d)"
+ cb6f404e1c3e9f0239ecde095c04dcb50a164ce346046f077744b2c53fab7fc4
+ b689d1d66a39a79b9db89e620061e910f9e2a49b9b1c06c58323b926ac61a658
+ $(vopt_if i3 432fb223a52927a32c75cb059ab79d263017918826f941edc9c9166a34832d09)"
 
 build_options="alsa curl i3 mpd network"
 build_options_default="$build_options"
 
-if [ "$build_option_network" -a "$XBPS_TARGET_MACHINE" = "aarch64-musl" ]; then
-	CXXFLAGS+=" -D_LINUX_IF_ETHER_H"
+desc_option_alsa="Enable support for ALSA"
+desc_option_curl="Enable support for cURL"
+desc_option_i3="Enable support for i3"
+desc_option_mpd="Enable support for MPD"
+desc_option_network="Enable support for network"
+
+CXXFLAGS="-Wno-error=deprecated-declarations"
```

Comment:
Isn't this already in https://github.com/jaagr/i3ipcpp/commit/d54efb9b76bd0abeb7ac99db7e2d95909306e899?

Reply:
I was not the one who added this to the xbps template so I don't know the reason for it. However, it's part of the merged template.